### ISSUE TITLE
feature: SBC to handle refer locally

### ISSUE THREAD

**Issue by vinoddh-kore** [jambonz/jambonz-feature-server#255]
It would be good to have a feature in which refer can be handled at Jambonz sbc rather than sending it source. There can be pattern configured based on refer-to

**Comment by davehorton** [comment]
Agree, for clarity this is an incoming REFER received by jambonz, as for example sent by a SIP client

**Comment by vinoddh-kore** [comment]
This Sipclient is connected to jambonz using register or sip refer originated by bot
